Logger/LogFactory.cs

```diff
@@ -1,11 +1,24 @@
-﻿namespace Logger
-{
+﻿namespace Logger;
+
     public class LogFactory
     {
-        public BaseLogger CreateLogger(string className)
+        private string? _filePath;
+
+        public BaseLogger? CreateLogger(string className)
         {
-            
-            return null;
+            if(_filePath is null)
+            {
+                return null;
+            }
+            else
+            {
+                return new FileLogger(_filePath) { ClassName = className };
+            }
```

Comment:
Looks great, if you'd like to get rid of even more code you could do this:
```suggestion
            if(_FilePath is null) { return null; }
            return new FileLogger(_FilePath) { ClassName = className };
```